testing/tests/DevExpress.ui.widgets.scheduler/navigator.tests.js

```diff
@@ -26,6 +26,15 @@ QUnit.module("Navigator", {
     }
 });
 
+QUnit.test("customizeDateNavigatorText should have right context", function(assert) {
+    var date = new Date(2018, 11, 14, 9, 20);
+
+    this.instance.option("date", date);
+    this.instance.option("customizeDateNavigatorText", function() {
+        assert.deepEqual(this, window, "context is ok");
```

Comment:
strictEqual?

Reply:
Actually, I'd prefer the expected value to be a custom context bounded on the handler.